scripts/measure_xsec_lumi.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+from ROOT import gROOT
+gROOT.SetBatch(True)
+import ROOT
+
+file_path = "/storage/epp2/phshgg/DVTuples__v23/5TeV_2017_32_Down_EW.root"
+
+
+#CALCULATE LUMINOSITY
+rfile = ROOT.TFile(file_path)
+lumi_tuple = rfile.Get('GetIntegratedLuminosity/LumiTuple')
+
+total_lumi = 0.0
+#total_lumi_err_sum = 0.0
+
+for entry in lumi_tuple:
+    #print(entry.IntegratedLuminosity, entry.IntegratedLuminosityErr)
+    total_lumi += entry.IntegratedLuminosity
+    #total_lumi_err_sum += entry.IntegratedLuminosityErr*entry.IntegratedLuminosityErr
+
+#Calculate error using tuple (not used)
+#total_lumi_err = total_lumi_err_sum**0.5
+    
+#Calculate error as 5%
+total_lumi_err = total_lumi*0.05
+
+print('Integrated Luminosity (pb-1) = {}'.format(total_lumi))
+print('IL 5% error (pb-1) = {}'.format(total_lumi_err))
+
+
+# CALCULATE CROSS SECTION
+ch = ROOT.TChain('Z/DecayTree')
+ch.Add(file_path)
+
+count = 0
+
+for entry in ch:
```

Comment:
Here it would be more efficient to use `ROOT`'s internal `TTree::GetEntries()` function. I think it is also clearer and more portable to write a cut string.
```python
data_yield = ch.GetEntries("Z_M > 60.e3 && Z_M < 120.e3 && ... etc...")
```